synapse/lib/stormlib/cortex.py

```diff
@@ -1317,3 +1318,47 @@ async def execStormCmd(self, runt, genr):
         await self.runt.printf(f'Storm Pool URL: {url}')
         await self.runt.printf(f'Sync Timeout (secs): {opts.get("timeout:sync")}')
         await self.runt.printf(f'Connection Timeout (secs): {opts.get("timeout:connection")}')
+
+@s_stormtypes.registry.registerLib
+class CortexApi(s_stormtypes.Lib):
+    '''
+    Library for interacting with the Cortex API.
+    '''
+    _storm_locals = (
+        {'name': 'getIdenByNid', 'desc': 'Get the iden for a node by its node id in this Cortex.',
+         'type': {'type': 'function', '_funcname': 'getIdenByNid',
+                  'args': (
+                      {'name': 'nid', 'type': 'int', 'desc': 'The node id of the node.'},
+                  ),
+                  'returns': {'type': 'str', 'desc': 'The iden of the node or None if the node id is not found.'}}},
+        {'name': 'getNidByIden', 'desc': 'Get the node id for an iden in this Cortex.',
+         'type': {'type': 'function', '_funcname': 'getNidByIden',
+                  'args': (
+                      {'name': 'iden', 'type': 'str', 'desc': 'The iden of the node.'},
+                  ),
+                  'returns': {'type': 'int', 'desc': 'The node id or None if the iden is not found.'}}},
+    )
+
+    _storm_lib_path = ('cortex',)
+
+    def getObjLocals(self):
+        return {
+            'getIdenByNid': self.getIdenByNid,
+            'getNidByIden': self.getNidByIden,
```

Comment:
Do we want to also add a `getNodeByNid()` API here that uses the current view so control-flow use cases can be simplified as well?